addons/project_issue_sheet/project_issue_sheet.py

```diff
@@ -25,8 +25,30 @@
 class project_issue(osv.osv):
     _inherit = 'project.issue'
     _description = 'project issue'
+
+    def _hours_get(self, cr, uid, ids, field_names, args, context=None):
+        res = {}
+        for issue in self.browse(cr, uid, ids, context=context):
+            progress = 0.0
+            if issue.task_id:
+                progress = issue.task_id.progress
+            res[issue.id] = {'progress' : progress}
+        return res
+
+    def _get_issue_task(self, cr, uid, task_ids, context=None):
```

Comment:
return self.pool['project.issue'].search(cr, uid, [('task_id', 'in', task_ids)], context=context)